### ISSUE TITLE
Concrete ContinousSpaceDF

### ISSUE THREAD

**Issue by adamamer20** [adamamer20/mesa-frames#36]

**Comment by luca-patrignani** [comment]
The reasonable way to solve implement a ContinuosSpaceDF is storing agents in [GeoPolars](https://github.com/geopolars/geopolars) dataframes, but the project is still not production-ready.

**Comment by adamamer20** [comment]
Yeah this is a bit tricky to do it without that. We could still use a `geopandas.GeoSeries` as a separate object but it requires more effort.
